index.html

```diff
@@ -274,10 +278,39 @@
                 this.centerv();
                 this.centerh();
                 return this;
-            }            
-
+            }
             
             $(document).ready(function() {
+                // instantiate Cionic SDK
```

Comment:
Interesting that the original repo just has all this JS and CSS inline in the HTML. If we end up doing more mods on this one, it might be worth breaking it up.